### PR TITLE
refactoring WebWorkerMainTemplatePlugin.test.js to ES6

### PR DESCRIPTION
**Pull request by sendilkumarn**
**What kind of change does this PR introduce?**
Refactor ES5 => ES6

### CODE REVIEW

File: test/WebWorkerMainTemplatePlugin.test.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+const applyPluginWithOptions = require("./helpers/applyPluginWithOptions");
 
 describe("WebWorkerMainTemplatePlugin", function() {
-	var env;
+	let env;
 
-	beforeEach(function() {
-		env = {};
+	beforeEach(() => {
+		env = {}
```
Comment (abouthiroppy) [inline test/WebWorkerMainTemplatePlugin.test.js]: Why did you delete the semicolon?

Comment (sendilkumarn) [inline test/WebWorkerMainTemplatePlugin.test.js]: tried to replace it with complete arrow function removing `{` and then decided against it. The effect of that :disappointed:

**Comment by sokra** [comment]
Thanks
